deenurp/subcommands/rdp_extract_genbank.py

```diff
@@ -114,9 +133,12 @@ def action(a):
             rdp_lineage = ';'.join(record.annotations.get('taxonomy', []))
             rdp_lineage = rdp_lineage.replace('"', '')
 
+            is_class = is_classified(tax_id)
+            is_class &= 'uncultured bacterium' not in get_organism(record)
```

Comment:
Does checking for 'uncultured bacterium' in the organism name this have any value here if we have already filtered on the basis of tax_id?